problem10.py

```diff
@@ -0,0 +1,60 @@
+#The Sieve of Erastothenes
+import math
+from get_number_from_index import get_number_from_index
+
+
+def get_idx_from_number(number):
+    index = (number-3) >> 1
+    return index
+
+
+
```

Comment:
In python, it is good to leave two new lines between functions. Not more or less. 2 is the number of spaces that should be between functions, not 1 which is too small, and not 3 which is too big.

Reply:
Please change it also in other files